homeassistant/components/geo_json_events/geo_location.py

```diff
@@ -104,7 +104,11 @@ async def async_update(self) -> None:
 
     def _update_from_feed(self, feed_entry: GenericFeedEntry) -> None:
         """Update the internal state from the provided feed entry."""
-        self._attr_name = feed_entry.title
+        self._attr_name = (
+            feed_entry.properties.get("name")
+            if feed_entry.properties and "name" in feed_entry.properties
+            else feed_entry.title
+        )
```

Comment:
Please dont use inline ternaries when they span more than one line

Reply:
It's required to be multi line as part of the linting process.

Reply:
It's required to be multi line as part of the linting process. Would something like the following be better?

```
        if feed_entry.properties and "name" in feed_entry.properties:
            self._attr_name = feed_entry.properties.get("name")
        else:
            self._attr_name = feed_entry.title
```

Reply:
I have amended from your feedback and all tests passed again.

Reply:
I have added this and added tests to cover that scenario too. All tests passed.